giskard/rag/evaluate.py

```diff
@@ -76,17 +80,27 @@ def evaluate(
     # @TODO: improve this
     metrics = metrics or []
     if not any(isinstance(metric, CorrectnessMetric) for metric in metrics):
-        metrics.append(CorrectnessMetric(name="Correctness", agent_description=agent_description))
+        # By default only correctness is computed as it is required to build the report
+        metrics.insert(
```

Comment:
just noticed this: `metrics` needs to be a `list` or casted with `list(metrics)`, not a generic sequence